cmd/logger.go

```diff
@@ -154,36 +161,42 @@ func logIf(level logrus.Level, source string, err error, msg string, data ...int
 	if err == nil || isErrIgnored(err) {
 		return
 	}
-
-	fields := logrus.Fields{
-		"source": source,
-		"cause":  err.Error(),
-	}
-
-	if terr, ok := err.(*errors.Error); ok {
-		fields["stack"] = strings.Join(terr.Stack(), " ")
+	var stack []string
+	cause := err.Error()
+	// Get stack trace
+	stack = trace(3)
+	// Get time
+	time := time.Now().UTC().Format(time.RFC3339Nano)
+	// Print formatted log message on the console
+	stackTab := "       "
+	levelCaps := strings.ToUpper(level)
+	message := fmt.Sprintf(msg, data...)
+	if log.json {
+		logJSON, _ := json.Marshal(&logEntry{
```

Comment:
handle `json.Marshal()` error i.e. `panic()` for error here.

Reply:
Done.